features/step_definitions/reek_todo/generate_todo_list_steps.rb

```diff
@@ -0,0 +1,37 @@
+Given(/^a directory 'lib' with one clean file 'clean\.rb' and one dirty file 'dirty\.rb'$/) do
+  write_file('lib/clean.rb', <<-EOS.strip_heredoc)
```

Comment:
No, because that would limit us to 2.3. And right now we're supporting all rubies [up from 2.0](https://github.com/troessner/reek/blob/master/.travis.yml#L8)